### PR TITLE
Convert manifest to yaml

### PR DESCRIPTION
**Pull request by cpba**

### CODE REVIEW

**Comment by cpba** [comment]
bot build

File: im.srain.Srain.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        url: https://www.imagemagick.org/download/releases/ImageMagick-7.0.6-10.tar.xz
+        sha256: 5a9889c87dc351cf4b57ca3ae418c9cdcefc77aaa9a3b16044ae8a4eecf3aeed
+      cleanup:
+        - "*"
+
+  - name: requests
+    buildsystem: simple
+    build-commands:
+      - "/usr/bin/pip3 --disable-pip-version-check install --prefix=/app --no-deps --verbose
+        ."
```
Comment (TingPing) [inline im.srain.Srain.yaml]: Spreading these out on two lines seems overkill

File: im.srain.Srain.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/source/i/idna/idna-2.6.tar.gz
+        sha256: 2c6a5de3089009e3da7c5dde64a141dbc8551d5b7f6cf4ed7c2568d0cc520a8f
+
+  - name: srain
+    config-opts:
+      - "--prefix=/app"
```
Comment (TingPing) [inline im.srain.Srain.yaml]: This is redundant.

File: im.srain.Srain.yaml (hunk 1 above)
Comment (cpba) [reply im.srain.Srain.yaml]: Indeed!

**Comment by cpba** [comment]
bot, build

**Comment by cpba** [comment]
I asked @SilverRainZ on irc and he approves of the change, I'm merging.

**Pull request merged**